content/en/security/application_security/guide/manage_account_theft_appsec.md

```diff
@@ -0,0 +1,669 @@
+
+### Step 1.2: Enabling ASM on your login service
+
+To enable ASM on your login service, ensure you meet the following requirements:
+
+* Similarly to Datadog APM, ASM requires a library integration in your services and a running Datadog agent.  
+* ASM generally benefits from using the newest library possible; however, minimum supported versions are documented in [Compatibility Requirements][3].   
+* **Threat Detection** is required at a minimum, and **Automatic user activity event tracking** should also be enabled, ideally.
+
+To enable ASM using a new deployment, use the `APPSEC\_ENABLED` environment variable/library configuration or [Remote Configuration]. You can use either method, but Remote Configuration can be set up using the Datadog UI.
+
+To enable ASM using Remote Configuration, do the following:
+
+1. Go to [Remote Configuration][5].  
+2. Click **Get Started with ASM**.   
+3. In **Threat Management**, click **Select Services.**   
+4. Select your service(s), and then click **Next** and proceed with the setup instructions.
+
+When you see traces from your service in [ASM Traces][6], move to [Step 1.3: Validating login information is automatically collected](#step-1.3:-validating-login-information-is-automatically-collected).
+
+For more detailed instructions on using a new deployment, see [Enabling ASM Threat Detection using Datadog Tracing Libraries][7].
+
+### Step 1.3: Validating login information is automatically collected {#step-1.3:-validating-login-information-is-automatically-collected}
+
+After you have enabled ASM, you can validate that login information is collected by Datadog.
+
+**Note:** Once ASM is enabled on a service, wait a few minutes for users to log into the service or log into the service yourself.
+
+To validate login information is collected, do the following:
+
+1. Go to [Traces][8] in ASM.   
+2. Look for traces tagged with login activity from your login service. For example, in **Search for**, you might have `@appsec.security\_activity:business\_logic.users.login.\*`.  
+3. Check if all your login services are reporting login activity. You can see this in the **Service** facet.
+
+<!-- ![][image4] -->
+
+**If you don't see login activity from a service**, go to [Step 1.5: Manually instrumenting your services](#step-1.5:-manually-instrumenting-your-services).
+
+### Step 1.4: Validating login metadata is automatically collected {#step-1.4:-validating-login-metadata-is-automatically-collected}
+
+To validate that login metadata is collected, do the following:
+
+1. Go to [Traces][8] in ASM.   
+2. Look for traces tagged with successful and failed login activity from your login service. For example, in **Search for**, you might have all.  
+3. Open a trace.  
+4. In the trace details, is the **Security** tab, review **Business Logic Event**.  
+5. Check if the event has a false user.
+
+<!-- ![][image5] -->
+
+In the event of a **false** user (`usr.exists:false`), look for the following issues:
```

Comment:
```suggestion
Review a few traces, both login success and login failures. In the case of login failures, look for traces with `usr.exists` both to true (failed login attempt on an existing user) and to false.
```
Those checks have to be done whether the user exist or not

Reply:
I left this last sentence in because it's needed to intro the bullet points:

> In the event of a **false** user (`usr.exists:false`), look for the following issues: